### DISCUSSION TITLE
Move and/or adjust columns

### DISCUSSION THREAD

**Discussion by dumpbox76** [spotweb/spotweb#876] [Announcements]
**Is your feature request related to a problem? Please describe.**
It's not a technical problem, it's a quality of life feature request.

My SpotWeb NZB button is connected to a SabNZB instance, and also using NZBGet. 
From Spotweb spot overview screen (main screens), I use the SabNZB and NZB button a lot.
However, when the computer screens are getting wider, the distance of those 2 buttons on the right is ever-increasing. And my eyes aren't getting younger as well ;)

**Describe the solution you'd like**
Would like to have the option to change the columns, so I could move and place the SaNZB + NZB button to the far left, next to the spot title.
Maybe make it also possible to adjust the column width ? Some long names can mess up the line distances, as showed in my screenshot.

**Describe alternatives you've considered**
None found.

**Additional context**
Screenshot included.
![spotwebrq](https://github.com/user-attachments/assets/d48ed15e-c8be-4d75-bfb3-26442ee811f0)

**Comment by mesa57** [comment]
Spotweb makes use of an pretty old template system which does not allow to move the columns or set the width,
However, you can copy the template (we1rdo) to a new own set and make you're desired changes there.
